probe-rs/src/architecture/riscv/communication_interface.rs

```diff
@@ -322,23 +317,6 @@ impl RiscvCommunicationInterface {
 
         self.state.implicit_ebreak = status.impebreak();
 
-        // check if the configuration string pointer is valid, and retrieve it, if valid
-        self.state.confstrptr = if status.confstrptrvalid() {
```

Comment:
Why was this removed?

Reply:
Hmm yeah good question how we want to handle different specs. Maybe we should treat it as a non issue until it becomes one.